### DISCUSSION TITLE
Can i use the seq logo for a home automation plugin?

### DISCUSSION THREAD

**Discussion by o0shojo0o** [datalust/seq-tickets#1247] [Help] [answered]
Hi, I wanted to find out to what extent I can use the seq logo for a home automation plugin.
Can someone give me some information about this? 

Greeting
Shojo

**Comment by nblumhardt** [comment, marked as answer]
Hi @o0shojo0o, thanks for getting in touch!

Using it to link to the Seq website is fine; or, if you're using it to promote integration between another product and Seq, this is also fine, as long as it's clear that the logo is representing an integration, and not the product in itself.

It's tricky to give a more concrete reply without knowing what your plug-in is/does, so please feel free to email me some more detail (my username, at datalust.co), if you would like to clarify anything further.

Hope this helps,
Nick
